.changeset/late-houses-work.md

```diff
@@ -0,0 +1,8 @@
+---
+'@leafygreen-ui/form-field': patch
+'@leafygreen-ui/date-picker': patch
```

Comment:
I don't think that you need to include date-picker, text-area, and text-input since you're not manually bumping the dependency version. The version PR should automatically bump the version and add a changeset.